### PR TITLE
remove dead code and clean up generated comments

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/grifts/db.go [hunk 1]
```diff
@@ -21,16 +21,9 @@ var _ = grift.Namespace("db", func() {
 		}
 
 		// ORGANIZATIONS Table
-		//organizationUUID1, _ := uuid.FromString("f3a79b30-f00e-48a0-a64d-e27748dea22d")
-		organizationUUID2, _ := uuid.FromString("d2e95724-9270-4050-82d9-6a9f9c35c766")
+		organizationUuid2, _ := uuid.FromString("d2e95724-9270-4050-82d9-6a9f9c35c766")
```
Comment (Baggerone) [inline application/grifts/db.go]: Did this get changed by something automated?  In any case, it doesn't match line 29 or the all-caps "rule".

Comment (Baggerone) [reply application/grifts/db.go]: Well, it's more of a "guideline"

Comment (Baggerone) [reply application/grifts/db.go]:
```suggestion
		organizationUUID2, _ := uuid.FromString("d2e95724-9270-4050-82d9-6a9f9c35c766")
```

Comment (Schparky) [reply application/grifts/db.go]: I got reverted because I missed it in resolving a merge conflict.

**Pull request merged**